LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Generation Software
```

Comment:
I've noticed this in the others as well, but do we want to say "Generation Software" or "G9 Software Inc."? The latter is the legal name, right? Does G9 Software Inc. officially have the "Generation Software" alias?

*(disclaimer: I'm not a lawyer and this isn't financial advice :3)*

Reply:
Yes; technically the company name is G9 Software Inc.

I can change this, and we can keep it in mind elsewhere